frontend/packages/portal-frontend/src/resources/components/ResourcesPage.tsx

```diff
@@ -52,43 +53,41 @@ export default function ResourcesPage(props: ResourcesPageProps) {
       </div>
 
       <section className={styles.postsNavList}>
-        {subcategories.map((subcategory: any) => {
-          return (
-            <Accordion key={subcategory.id} title={subcategory.title}>
-              <ListGroup style={{ marginBottom: 0, borderRadius: 0 }}>
-                {subcategory.topics.map((topic: any) => {
-                  return (
-                    <Link
-                      key={topic.id}
-                      to={`?subcategory=${subcategory.slug}&topic=${topic.slug}`}
-                      state={{ postHtml: topic }}
-                      style={{ textDecoration: "none" }}
-                    >
-                      <ListGroupItem
-                        className={styles.navPostItem}
-                        style={{ borderRadius: "0px" }}
-                        active={initPost ? initPost.slug === topic.slug : false}
-                      >
-                        {topic.title}
-                      </ListGroupItem>
-                    </Link>
-                  );
-                })}
-              </ListGroup>
-            </Accordion>
-          );
-        })}
+        <PanelGroup>
```

Comment:
That's fair! You're right that I probably don't need PanelGroup. 

I think I initially used it because I noticed it was how they were grouping Panel components. I thought it would be useful to use their toggle props but I believe `activeKey` only toggle opens one accordion at a time while closing the other that doesn't match the `activeKey` (correct me if I'm wrong). I wanted to be able to have more than one panel/accordion open at the same time which is why I made `Panel` into a controlled component instead in `SubcategoryPanel.tsx`

Reply:
I see, that makes sense! You are correct about activeKey.